src/pages/home/report/ReportActionsView.js

```diff
@@ -31,13 +31,13 @@ import EmojiPicker from '../../../components/EmojiPicker/EmojiPicker';
 import * as ReportActionsUtils from '../../../libs/ReportActionsUtils';
 
 const propTypes = {
-    /** The ID of the report actions will be created for */
-    reportID: PropTypes.number.isRequired,
-
     /* Onyx Props */
 
     /** The report currently being looked at */
     report: PropTypes.shape({
+        /** The ID of the report actions will be created for */
+        reportID: PropTypes.number,
```

Comment:
Can you also make this one required?
```suggestion
        reportID: PropTypes.number.isRequired,
```

I think it's the only property that is required for a report.